packages/blade/src/components/LiveAnnouncer/LiveAnnouncer.web.tsx

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+// Modified version of @react-aria/live-announcer
+// https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/live-announcer/src/LiveAnnouncer.tsx
+// LiveAnnouncer is implemented using vanilla DOM, not React.
+// While the spectrum team had their own reasons for using vanillajs,
+// For us:
+// 1. It's significantly easier to maintain & understand LiveAnnouncer code in vanillajs than React.
+//    With React maintaining a singleton instance of LiveAnnouncer is tricky because we need
+//    to save the instance of ReactElement to a top level ref, and then use ReactDOM.render() to render it
+// 2. Providing imperative APIs like `announce()` is also trikcky which involves using lesser known
+//    React hook `useImperativeHandle` to give users imperative access.
+
+import screenReaderStyles from '../VisuallyHidden/ScreenReaderStyles';
+import type { Assertiveness } from './LiveAnnouncer.d';
+
+const LIVEREGION_TIMEOUT_DELAY = 1000;
+let liveAnnouncer: LiveAnnouncer | null = null;
+
+class LiveAnnouncer {
+  node: HTMLElement;
+  assertiveLog: HTMLElement;
+  politeLog: HTMLElement;
+
+  constructor() {
+    this.node = document.createElement('div');
+    this.node.dataset.liveAnnouncer = 'true';
+    Object.assign(this.node.style, screenReaderStyles);
+
+    this.assertiveLog = this.createLog('assertive');
+    this.node.appendChild(this.assertiveLog);
+
+    this.politeLog = this.createLog('polite');
+    this.node.appendChild(this.politeLog);
+
+    document.body.prepend(this.node);
+  }
+
+  createLog(ariaLive: string): HTMLDivElement {
+    const node = document.createElement('div');
+    node.setAttribute('role', 'log');
+    node.setAttribute('aria-live', ariaLive);
+    node.setAttribute('aria-relevant', 'additions');
+    return node;
+  }
+
+  destroy(): void {
+    if (!this.node) {
+      return;
+    }
+
+    document.body.removeChild(this.node);
+    // @ts-expect-error
+    this.node = null;
+  }
+
+  announce(message: string, assertiveness = 'assertive', timeout = LIVEREGION_TIMEOUT_DELAY): void {
```

Comment:
> shall we make the input args same as native? since they are same functions just diff platforms

I've made the function signature is already same between native & web. 

> can we convert the input args to {}, we are following this whenever we have a function that has more than 1 arg

More than 1? or more than or eq to 2?, 1 arg per function seems a bit overkill tho, The standard pattern I've seen is more than 3 or even 2 (unless the last argument is a boolean flag)

In this case announce in general will only take 1 args for most cases `message` 
Both the `assertive` & `timeout` is optional and won't be need mostly, and TS is there to inform the signature while using.